packages/renderer/src/lib/dashboard/release-notes/ReleaseNotesBox.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+import { faCircleArrowUp } from '@fortawesome/free-solid-svg-icons';
+import { Button, Link } from '@podman-desktop/ui-svelte';
+import { onMount } from 'svelte';
+
+let updateAvilable = false;
+let currentVersion: string;
+let content: HTMLElement | undefined;
+let title: string = '';
+let summaryTitle = '';
+let summary: HTMLUListElement;
+let imageUrl: string = '';
+let imageAlt: string = '';
+
+async function getImageFromNotes() {
+  currentVersion = '1.12.0';
+  let curVersionSplit = currentVersion.split('.', 2);
+  const urlVersionFormat = curVersionSplit.join('.');
+  let notesURL = `https://podman-desktop.io/blog/podman-desktop-release-${urlVersionFormat}`;
+  const response = await fetch(notesURL);
+  if (!response.ok) {
```

Comment:
nice improvement

here we might need to use the source file rather than the published one (open question)

https://raw.githubusercontent.com/containers/podman-desktop/main/website/blog/2024-08-08-release-1.12.md

(we might need a json file with the mapping between the version and the source file or another pattern to find the file)

but then we have a file with all the data

also `<!--truncate-->` needs to be defined to know when to split the full content

Reply:
@benoitf my first thought was also to use the raw md github file instead of the website, but having the date in the link makes it more challenging since we cannot guess it automatically like the version in the link. Since this file might not even exists when the binary is cut, how can we add it to a mapping that is included in the app before the release? We could add a mapping with some specific date/file name before the release and rely on the notes' author to use the same date/name for the file.

Reply:
I've opened a PR for the mapping, and the new commit I pushed shows how I'm planning on fetching the information from the raw files. Right now, you can see the difference between using the website's notes (`ReleaseNotesBox`) and GitHub's notes (`ReleaseNotesBoxCopy`). `ReleaseNotesBoxCopy` has a placeholder notes link that I'll change once https://github.com/containers/podman-desktop/pull/8790 is merged